packages/cells/test/model.spec.ts

```diff
@@ -541,6 +542,111 @@ describe('cells/model', () => {
       });
     });
 
+    describe('#onModelDBOutputsChange()', () => {
+      const output0 = {
+        output_type: 'display_data',
+        data: {
+          'text/plain': 'foo',
+          'application/json': { foo: 1 }
+        },
+        metadata: {}
+      } as nbformat.IDisplayData;
+      const output1 = {
+        output_type: 'display_data',
+        data: {
+          'text/plain': 'bar',
+          'application/json': { bar: 2 }
+        },
+        metadata: {}
+      } as nbformat.IDisplayData;
+      const output2 = {
+        output_type: 'display_data',
+        data: {
+          'text/plain': 'foobar',
+          'application/json': { foobar: 2 }
+        },
+        metadata: {}
+      } as nbformat.IDisplayData;
+      const cell: nbformat.ICodeCell = {
+        cell_type: 'code',
+        execution_count: 1,
+        outputs: [output0, output1],
+        source: 'foo',
+        metadata: { trusted: false },
+        id: 'cell_id'
+      };
+      it('should add new items correctly', () => {
+        const model = new CodeCellModel({});
+        const sharedModel = model.sharedModel as YCodeCell;
+        expect(sharedModel.ymodel.get('outputs').length).toBe(0);
+
+        const newEvent0 = {
+          type: 'add',
+          newValues: [{ toJSON: () => output0 }],
+          oldValues: [],
+          oldIndex: -1,
+          newIndex: 0
+        } as any;
+        model['onModelDBOutputsChange'](null as any, newEvent0);
```

Comment:
Is there a reason for using a key getter and not

```suggestion
        model.onModelDBOutputsChange(null as any, newEvent0);
```

Reply:
Hello @fcollonval , it's because `onModelDBOutputsChange` is a protected method.

Reply:
thanks - I didn't look at the code :wink: